edx_name_affirmation/__init__.py

```diff
@@ -2,4 +2,4 @@
 Django app housing name affirmation logic.
 """
 
-__version__ = '2.3.7'
+__version__ = '2.4.0'
```

Comment:
@MichaelRoytman @UsamaSadiq Could you bump this version on pip? Please. I need to update [edx-opaque-keys](https://github.com/edx/edx-name-affirmation/pull/189/files#diff-0940fec0afe21798f617cc4522db676f15769ae28a72992b8d19216ef7b1c3b5R104) in other repos. Thanks!

Reply:
@ChrisChV Thank you for the ping. I just published a new release. It should be available on PyPI! Please let me know if you need anything else.

Reply:
Thanks!!